datamodel/src/test/java/com/hartwig/serve/datamodel/DatamodelTestFactory.java

```diff
@@ -13,22 +13,69 @@ public final class DatamodelTestFactory {
     private DatamodelTestFactory() {
     }
 
+    @NotNull
+    public static ImmutableClinicalTrial.Builder clinicalTrialBuilderBuilder() {
+        return ImmutableClinicalTrial.builder()
+                .studyNctId("studyNctId")
+                .studyTitle("studyTitle")
+                .countriesOfStudy(Sets.newHashSet("countriesOfStudy"))
+                .therapyName("therapyName");
+    }
+
```

Comment:
For test builders I would suggest to set minimal values (either null or Strings.EMPTY)